packages/data-store/src/api/user.js

```diff
@@ -65,10 +65,9 @@ export default {
       postcode: user.postalCode,
       name: user.name,
       fullName: user.name,
-      displayName: user.name,
+      displayName: user.nickName,
```

Comment:
Zit er niet een `user.displayName` in de API? Want die bevat de nickname / de naam als de nickname niet bestaat. Dan hoeven we die logica hier niet meer in te bouwen.